src/test/java/com/epam/tc/hw2/ex1/Ex1.java

```diff
@@ -0,0 +1,174 @@
+        WebElement enterButton = driver.findElement(By.id("login-button"));
+        enterButton.click();
+
+        //4. Assert Username is loggined
+        WebElement userName = driver.findElement(By.id("user-name"));
+        System.out.println(userName.getText());
+        softly.assertThat(userName.getText()).as("Incorrect user name").isEqualTo("ROMAN IOVLEV");
+
+        //5. Assert that there are 4 items on the header section are displayed and they have proper texts
+        WebElement homeButton = driver.findElement(By.cssSelector("a[href='index.html']"));
+        System.out.println(homeButton.getText());
+        softly.assertThat(homeButton.getText()).as("Incorrect Home button name").isEqualTo("HOME");
+
+        WebElement contactFormButton = driver.findElement(By.cssSelector("a[href='contacts.html']"));
+        softly.assertThat(contactFormButton.getText())
+                .as("Incorrect Contacts button name")
+                .isEqualTo("CONTACT FORM");
+
+        WebElement serviceButton = driver.findElement(By.cssSelector("a.dropdown-toggle"));
+        softly.assertThat(serviceButton.getText()).as("Incorrect Service button name").isEqualTo("SERVICE");
+
+        WebElement metalsColorsButton = driver.findElement(By.cssSelector("a[href='metals-colors.html']"));
+        softly.assertThat(metalsColorsButton.getText())
+                .as("Incorrect METALS & COLORS button name").isEqualTo("METALS & COLORS");
+
+        //6. Assert that there are 4 images on the Index Page and they are displayed
+        List<WebElement> icons = driver.findElements(By.cssSelector(".benefit-icon"));
+        softly.assertThat(icons.size()).as("Number of images are " + icons.size() + " not 4").isEqualTo(4);
+        softly.assertThat(icons).as("One or more image not displayed").isNotEmpty();
+        //other way: assertTrue(isImageDisplayed);
+
+        //7. Assert that there are 4 texts on the Index Page under icons and they have proper text
+        List<WebElement> iconTexts = driver.findElements(By.cssSelector(".benefit-txt"));
+        softly.assertThat(iconTexts.size())
+                .as("Number of texts are " + iconTexts.size() + " not 4").isEqualTo(4);
+        softly.assertThat(iconTexts.get(0).getText())
+                .as("To include good...text differs")
+                .isEqualTo("To include good practices\nand ideas from successful\nEPAM project");
+        softly.assertThat(iconTexts.get(1).getText())
+                .as("To be flexible...text differs").isEqualTo("To be flexible and\ncustomizable");
+        softly.assertThat(iconTexts.get(2).getText())
+                .as("To be multiplatform").isEqualTo("To be multiplatform");
+        softly.assertThat(iconTexts.get(3).getText())
+                .as("Already have good base...text differs")
+                .isEqualTo("Already have good base\n"
+                        + "(about 20 internal and\n"
+                        + "some external projects),\nwish to get more…");
```

Comment:
Honestly speaking there is a better way to do these checks.
You can using stream to map `iconTexts` to `List<String>`
And then compare it with expected `List<String>`
```
        List<String> textUnderImagesExpected = Arrays.asList(
                "1",
                "2",
                "3",
                "4"
        );
```
But it's not a requirement

Reply:
Fixed here and in step 11